src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java

```diff
@@ -124,7 +125,29 @@ public MesosCloud(
       boolean checkpoint,
       boolean onDemandRegistration,
       String jenkinsURL) throws NumberFormatException {
-    super("MesosCloud");
+    this("MesosCloud", nativeLibraryPath, master, description, frameworkName, slavesUser, 
```

Comment:
The only thing I see here is that this constructor has a lot of parameters and of similar type `String`. It is not bad but it can be easy for a developer that wants to use this class to misplaced some of the parameters during the instantiation. Maybe it could be interesting to create some kind of wrapper classes that groups the parameters for meaning, for example `Security`, `Slaves`, ...